lib/index.js

```diff
@@ -1001,6 +1001,22 @@ addToplevelMethod('of', function (x) {
     return _([x]);
 });
 
+/**
+ * Creates an empty stream.
+ *
+ * @id empty
+ * @section Utils
+ * @name _.empty()
+ * @returns Stream
+ * @api public
+ *
+ * _.empty().toArray(_.log); // => []
+ */
+
+addToplevelMethod('empty', function () {
+    return _([]);
```

Comment:
Use `this([])` instead of `_([])`.

There's a new feature in 3.0 (not yet documented) called `use` that lets you extend the stream prototype. Using `this` will make sure that you construct the new stream using the correct prototype.

Note that `of` has the same bug.